tests/Custom/TimeoutTest.php

```diff
@@ -64,4 +65,17 @@ public function testLongTimeoutWaitsForElementToAppear(): void
 
         $this->assertNotNull($element);
     }
+
+    public function testShortPageLoadTimeoutThrowsException(): void
+    {
+        $session = $this->getSession();
+        $driver = $session->getDriver();
+        \assert($driver instanceof WebdriverClassicDriver);
+
+        $driver->setTimeouts(array('page' => 1)); // Use 1ms timeout to avoid any successful page load.
+
+        $this->expectException('\Behat\Mink\Exception\DriverException');
+        $this->expectExceptionMessage('Page failed to load: ');
+        $session->visit('https://www.webpagetest.org/'); // Use external URL, because it loads slower, then local.
```

Comment:
I'm not sure I like this test:
1. it is very unlikely, but still remotely possible that the page loads in 1ms.
2. it depends on some arbitrary external site out of our control - all sorts of things might happen there.

May I suggest using our fixtures with a page that intentionally simulates a slow page load?
Something like so:
```php
<?php

// slow-page.php

echo 'Hello ';
sleep(2);
echo 'World';
```

That would mean putting this in the driver-testsuite, which I think would actually make sense and benefit other drivers.

Reply:
Done.